### PR TITLE
Change Learning Goal Structure

### LINKED ISSUE #41: Change learning goal structure
Instead of having multiple files for each learning goals, we should only have one file per chapter. This will reduce complexity when new curricula are created.

### PR DESCRIPTION
**Pull request by programming-wolf**
close #41

### CODE REVIEW

File: docs/01-basics/00-basics.adoc [hunk 1]
```diff
@@ -2,17 +2,7 @@
...
-== Grundlegendes
-
-// end::DE[]
-
-// tag::EN[]
-
-== Basics
-
-// end::EN[]
+== {basics}}
```
Comment (larsrh) [inline docs/01-basics/00-basics.adoc]: Double closing braces

Comment (larsrh) [inline docs/01-basics/00-basics.adoc]:
```suggestion
== {basics}
```

File: docs/01-module-block-1/00-structure.adoc [hunk 2]
```diff
@@ -2,19 +2,11 @@
...
 // ====================================================
 
-// tag::DE[]
-== Lerneinheit 1
-// end::DE[]
-
-// tag::EN[]
-== Lesson 1
-// end::EN[]
+== {lesson} 1
```
Comment (larsrh) [inline docs/01-module-block-1/00-structure.adoc:5]: Nitpick: "Lerneinheit 1" is bad German. Better would be "1. Lerneinheit".

File: docs/02-module-block-2/00-structure.adoc [hunk 3]
```diff
@@ -2,22 +2,11 @@
...
+== {lesson} 2
 
 include::01-duration-terms.adoc[{include_configuration}]
 
-
-=== {learning-goals}
-
-include::LZ-2-01.adoc[{include_configuration}]
-include::LZ-2-02.adoc[{include_configuration}]
+include::02-learning-goals.adoc[{include_configuration}]
```
Comment (larsrh) [inline docs/02-module-block-2/00-structure.adoc:9]: Much love for merging those files into a single one.

**Review by larsrh — DISMISSED** [review]
LGTM (modulo minor nitpicks)

File: docs/02-module-block-2/00-structure.adoc (hunk 3 above)
Comment (programming-wolf) [reply docs/02-module-block-2/00-structure.adoc:9]: 💜

File: docs/01-basics/00-basics.adoc (hunk 1 above)
Comment (programming-wolf) [reply docs/01-basics/00-basics.adoc]: Fixed, thx.

File: docs/01-module-block-1/00-structure.adoc (hunk 2 above)
Comment (programming-wolf) [reply docs/01-module-block-1/00-structure.adoc:5]: Cannot be changed easily at the moment. :(

**Review by larsrh — APPROVED** [review]

**Pull request merged**